resources/migrations/000_migrations.yaml

```diff
@@ -13005,6 +13005,83 @@ databaseChangeLog:
         - dbms:
             type: mysql,mariadb
 
+  - changeSet:
+      id: v46.00-001
+      author: tsmacdonald
+      comment: Added 0.46.0 -- Join table connecting cards to dashboards/cards that need custom filter values from the card
+      changes:
+        - createTable:
+            tableName: values_card
+            remarks: "Join table connecting cards to entities (dashboards, other cards, etc.) that use the values generated by the card for filter values"
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: updated_at
+                  type: ${timestamp_type}
+                  defaultValueComputed: current_timestamp
+                  remarks: "most recent modification time"
+                  constraints:
+                    nullable: false
+              - column:
+                  name: created_at
+                  type: ${timestamp_type}
+                  defaultValueComputed: current_timestamp
+                  remarks: "creation time"
+                  constraints:
+                    nullable: false
+              - column:
+                  name: card_id
+                  type: int
+                  remarks: "ID of the card generating the values"
+                  constraints:
+                    nullable: false
+              - column:
+                  name: parameterized_object_type
+                  type: varchar(255)
+                  remarks: "Type of the entity consuming the values (dashboard, card, etc.)"
+                  constraints:
+                    nullable: false
+              - column:
+                  name: parameterized_object_id
+                  type: int
+                  remarks: "ID of the entity consuming the values"
+                  constraints:
+                    nullable: false
+              - column:
+                  name: parameter_id
+                  type: varchar(255)
+                  remarks: "The parameter ID"
+                  constraints:
+                    nullable: false
+              - column:
```

Comment:
what's it for?
Is it mean to store `label-column-name and value-column-name`?


if it is, why don't we store it in the parameter itself?

Reply:
> Is it meant to store `label-column-name` and `value-column-name`?

Yes.

> why don't we store it in the parameter itself?

I want to leave the door open for querying by `parameter_id`, but more importantly I want the uniqueness constraint on the 4-tuple of `parameterized_object_id, parameterized_object_type, parameter_id, card_id`

Reply:
oops wrong line, my question is for `column_config` column.